tests/material-ui/check-slinky/m/material-ui/src/main/scala/typingsSlinky/materialUi/MaterialUI/BottomNavigation/BottomNavigationItemProps.scala

```diff
@@ -17,5 +17,31 @@ object BottomNavigationItemProps {
     if (className != null) __obj.updateDynamic("className")(className.asInstanceOf[js.Any])
     __obj.asInstanceOf[BottomNavigationItemProps]
   }
+  @scala.inline
+  implicit class BottomNavigationItemPropsOps[Self <: BottomNavigationItemProps] (val x: Self) extends AnyVal {
```

Comment:
If possible, try to generate a `private val x` instead of `val x` to avoid adding `x` as an extension.